cmd/kubeadm/app/apis/kubeadm/v1beta3/bootstraptokenstring.go

```diff
@@ -31,7 +31,7 @@ import (
 // of view and as an authentication method for the node in the bootstrap phase of
 // "kubeadm join". This token is and should be short-lived
 type BootstrapTokenString struct {
-	ID     string `json:"-" datapolicy:"token"`
+	ID     string `json:"-"`
```

Comment:
in reality token IDs are public, so i would consider this a bug in the tooling.
can it be fixed?

/hold

Reply:
oh, never mind....we are removing the token tag from the ID.
that seems correct in that case.

/hold cancel

Reply:
yeah, from the PR body:
> @SataQiu: in #102174 you added a datapolicy:"token" tag to the ID field of kubeadm's BootstrapTokenString. I would like to double check that this ID is indeed sensitive data that shouldn't be logged, because it is currently causing the analyzer to produce a finding.